Library/Homebrew/brew.sh

```diff
@@ -306,7 +313,17 @@ update-preinstall() {
 
   if [[ "$HOMEBREW_COMMAND" = "install" || "$HOMEBREW_COMMAND" = "upgrade" || "$HOMEBREW_COMMAND" = "tap" ]]
   then
+    if [[ -z "$HOMEBREW_VERBOSE" ]]
+    then
+      update-preinstall-timer & timer_pid=$!
+    fi
+
     brew update --preinstall
+
+    if [[ -n "$timer_pid" ]]
+    then
+      { kill "$timer_pid"; wait "$timer_pid"; } 2>/dev/null
```

Comment:
Ok. Can you split onto multiple lines? Does this need handled on `exit` or will that be done automatically by process cleanup?